Wikipedia/Code/PlacesViewController.swift

```diff
@@ -212,36 +208,43 @@ class PlacesViewController: PreviewingViewController, UISearchBarDelegate, Artic
         searchSuggestionController = PlaceSearchSuggestionController()
         searchSuggestionController.tableView = searchSuggestionView
         searchSuggestionController.delegate = self
-        
-        if UIAccessibilityIsVoiceOverRunning() {
-            viewMode = .list
-            mapListToggle.selectedSegmentIndex = 1
-        } else {
-            viewMode = .map
-        }
+
+        super.viewDidLoad()
 
         let panGR = UIPanGestureRecognizer(target: self, action: #selector(handlePanGesture))
         panGR.delegate = self
         view.addGestureRecognizer(panGR)
         overlaySliderPanGestureRecognizer = panGR
-        
-        apply(theme: theme)
+
         self.view.layoutIfNeeded()
     }
+
+    private var isFirstAppearance = true
```

Comment:
`viewWillHaveFirstAppearance` could probably move from `ColumnarCollectionViewController` up into `ViewController` so it could be re-used here